documentation/modules/openstack-prerequisites.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="openstack-prerequisites_{context}"]
+= OpenStack prerequisites
+
+The following prerequisites apply to {osh} migrations:
+
+* You must use a xref:compatibility-guidelines_{context}[compatible version] of {osh}.
+* You must have the {manager} CA certificate or the {manager} Apache CA certificate if the {manager} CA certificate was replaced by a third-party certificate on the Apache server.
```

Comment:
about the "You must have" - we'll support configuring an openstack provider without specifying a certificate (by selecting the option `Skip certificate validation (if checked, the provider's certificate won't be validated)`), so it's not a must-have

openstack doesn't have a central management unit, like rhv or vsphere, so we better avoid the use of "Manager" in this context. I'd also drop the `Apache` part - openstack is written in python and uses different tooling than RHV. in my opinion, it's ok to be a bit more vague here and say that the CA certificate for connecting to the source environment is needed, like we write for vSphere